web/src/hooks/arrows.ts

```diff
@@ -16,6 +16,30 @@ import exchangesToExclude from '../../config/excluded_aggregated_exchanges.json'
 
 // TODO: set up proper typed method for retrieving config files.
 const exchangesConfig: Record<string, any> = exchangesConfigJSON;
+const { exchangesToExcludeZoneView, exchangesToExcludeCountryView } = exchangesToExclude;
+
+export function filterExchanges(
+  exchanges: Record<string, ExchangeResponse>,
+  exclusionArrayZones: string[],
+  exclusionArrayCountries: string[]
+) {
+  const exclusionSetZones = new Set(exclusionArrayZones);
+  const exclusionSetCountries = new Set(exclusionArrayCountries);
+  const resultZones: Record<string, ExchangeResponse> = {};
+  const resultCountries: Record<string, ExchangeResponse> = {};
+  // Loop through the exchanges and assign them to the correct result object
+  for (const key of Object.keys(exchanges)) {
+    if (exclusionSetZones.has(key)) {
+      resultCountries[key] = exchanges[key];
+      continue;
+    }
+    if (exclusionSetCountries.has(key)) {
+      resultZones[key] = exchanges[key];
+    }
+  }
```

Comment:
```suggestion
  for (const [key, value] of Object.entries(exchanges)) {
    if (exclusionSetZones.has(key)) {
      resultCountries[key] = value;
      continue;
    }
    if (exclusionSetCountries.has(key)) {
      resultZones[key] = value;
    }
  }
```

Maybe a slight performance increase?